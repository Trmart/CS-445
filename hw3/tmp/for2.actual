.   .   .   .   Child: 0  Id: a [line: 5]
.   .   .   .   .   Child: 0  Id: a [line: 6]
.   .   .   .   .   .   Child: 0  Id: i [line: 9]
.   .   Child: 0  Var: a of array of type int [line: 3]
.   .   .   Child: 0  Var: i of type int [line: 5]
.   .   .   .   .   Child: 0  Var: i of type int [line: 7]
.   .   .   .   Child: 0  Var: j of type int [line: 6]
.   Child: 1  Compound [line: 2]
.   .   Child: 1  For [line: 5]
.   .   .   .   .   Child: 1  Id: i [line: 6]
.   .   .   .   Child: 1  Id: j [line: 5]
.   .   .   .   .   .   Child: 1  Id: j [line: 9]
.   .   .   .   .   Child: 1  Op: + [line: 9]
.   .   .   Child: 1  Range [line: 5]
.   .   .   .   Child: 1  Range [line: 6]
.   .   .   .   Child: 2  Compound [line: 6]
.   .   .   Child: 2  For [line: 6]
ERROR(8): Symbol 'j' is already declared at line 6.
ERROR(LINKER): A function named 'main()' must be defined.
Func: ( returns type void [line: 1]
Number of errors: 2
Number of warnings: 3
.   .   .   .   .   Sibling: 1  Var: j of type int [line: 8]
WARNING(3): The variable 'a' seems not to be used.
WARNING(5): The variable 'i' seems not to be used.
WARNING(9): Variable 'i' may be uninitialized when used here.
