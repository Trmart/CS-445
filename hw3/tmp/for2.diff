0a1,17
> .   .   .   .   Child: 0  Id: a [line: 5]
> .   .   .   .   .   Child: 0  Id: a [line: 6]
> .   .   .   .   .   .   Child: 0  Id: i [line: 9]
> .   .   Child: 0  Var: a of array of type int [line: 3]
> .   .   .   Child: 0  Var: i of type int [line: 5]
> .   .   .   .   .   Child: 0  Var: i of type int [line: 7]
> .   .   .   .   Child: 0  Var: j of type int [line: 6]
> .   Child: 1  Compound [line: 2]
> .   .   Child: 1  For [line: 5]
> .   .   .   .   .   Child: 1  Id: i [line: 6]
> .   .   .   .   Child: 1  Id: j [line: 5]
> .   .   .   .   .   .   Child: 1  Id: j [line: 9]
> .   .   .   .   .   Child: 1  Op: + [line: 9]
> .   .   .   Child: 1  Range [line: 5]
> .   .   .   .   Child: 1  Range [line: 6]
> .   .   .   .   Child: 2  Compound [line: 6]
> .   .   .   Child: 2  For [line: 6]
2,3c19,23
< Number of errors: 1
< Number of warnings: 3
---
> ERROR(LINKER): A function named 'main()' must be defined.
> Func: ( returns type void [line: 1]
> Number of errors: 2
> Number of warnings: 4
> .   .   .   .   .   Sibling: 1  Var: j of type int [line: 8]
6a27
> WARNING(9): Variable 'j' may be uninitialized when used here.
