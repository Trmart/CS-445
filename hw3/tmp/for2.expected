ERROR(8): Symbol 'j' is already declared at line 6.
Number of errors: 1
Number of warnings: 3
WARNING(3): The variable 'a' seems not to be used.
WARNING(5): The variable 'i' seems not to be used.
WARNING(9): Variable 'i' may be uninitialized when used here.
